src/routes/polls/polls.controller.js

```diff
@@ -1,11 +1,32 @@
 const db = require("./../../db/index");
 
-const postPolls = function (req, res) {
+const postPolls = function(req, res) {
   const body = req.body;
   console.log(body);
-  res.set('Content-Type', 'application/json')
+  res.set("Content-Type", "application/json");
   res.send(`Received new Poll: ${body.message}`);
-}
+
+  // Function to insert data
+  const insertData = (col, data) => {
+    const doc = db
+      .collection(col)
+      // Table
+      .doc();
+    const id = doc.id;
+    data["id"] = id;
+    return (
+      doc
+        // Data. .set returns a promise
+        .set(data)
+    );
```

Comment:
[style] I don't think you need the `()` here?

Reply:
Not sure what you mean by here? It's not a styling thing, it's the syntax to .set the data that we received

Reply:
I just meant you could do

```
return doc
    .set(data);
```

Reply:
Oh you mean remove the comment?

Reply:
just the bracket between `return` and `doc` haha...no big deal...